xml/System.Security.Permissions/RegistryPermissionAccess.xml

```diff
@@ -41,7 +41,7 @@
  <xref:System.Security.Permissions.RegistryPermissionAccess> values are independent; rights to one type of access do not imply rights to another. For instance, `Write` permission does not imply permission to `Read` or `Create`.  
   
 > [!NOTE]
->  Although NoAccess and AllAccess appear in <xref:System.Security.Permissions.RegistryPermissionAccess.NoAccess>, they are not valid for use as the parameter for <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A> because they describe no registry variable access types or all registry variable access types, respectively, and <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A> expects a single registry variable access type.  
+> Although `NoAccess` and `AllAccess` appear in <xref:System.Security.Permissions.RegistryPermissionAccess>, they are not valid for use as the parameter for <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A> because they describe no registry variable access types or all registry variable access types, respectively, and <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A> expects a single registry variable access type.  
```

Comment:
```suggestion
> Although `NoAccess` and `AllAccess` appear in `RegistryPermissionAccess`, they are not valid for use as the parameter for <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A?displayProperty=nameWithType> because they describe no registry variable access types or all registry variable access types, respectively, and <xref:System.Security.Permissions.RegistryPermission.GetPathList%2A> expects a single registry variable access type.  
```